build.gradle

```diff
@@ -28,12 +28,33 @@ intellij {
 dependencies {
     compile 'com.redhat.devtools.intellij:intellij-common:1.2.0-SNAPSHOT'
     testCompile 'org.mockito:mockito-inline:3.8.0'
+    compile 'com.squareup.okio:okio:1.15.0'
```

Comment:
What is the rationale for this ?

Reply:
To prevent the error with the f8 dependency. The same we have in tekton
![image](https://user-images.githubusercontent.com/49404737/120610810-d7fddb80-c453-11eb-938d-a9a0329808a4.png)